### ISSUE TITLE
Webhook does not work

### ISSUE THREAD

**Issue by guilhermeagirardi** [spatie/laravel-webhook-server#32]
Queue driver: `sync`
OS: `Windows 10 x64`
PHP: `7.4.2 TS`
Laravel: `v6.10.1`
laravel-webhook-server: `1.4.0`

Hello guys!

For some reason, the job is not performed. I create the Webhook and nothing happens, nor does anything appear in the log.

Is it because I'm using **Windows**?

My code:
```php
<?php

namespace App\Http\Controllers;

use Spatie\WebhookServer\WebhookCall;

class TestController extends Controller {
    public function index() {
        WebhookCall::create()
            ->url('https://app.test/hello')
            ->payload(['1' => '2'])
            ->withHeaders([])
            ->useSecret('test')
            ->dispatch();
    }
}
```

**Comment by freekmurze** [comment]
Our tests suite runs fine on windows: https://github.com/spatie/laravel-webhook-server/runs/406202504

I cannot reproduce this problem, so you'll have to debug this yourself. If it turns out that the package is in error, ping me here and I'll reopen the issue.

**Comment by guilhermeagirardi** [comment]
I discovered the problem. On line 62 of the `CallWebhookJob.php` file, an exception was thrown by `Guzzle`:
`cURL error 60: SSL certificate problem: unable to get local issuer certificate`

I checked the Job code, and realized that the exceptions thrown in this area are not printed in the log.

My suggestion is to put a `report($exception)` on line 77, this will greatly improve our lives, since if the errors are not logged, then we have the impression that the task was successfully executed, because there is nothing in the log.

Pull request: #33
